1-js/13-modules/03-modules-dynamic-imports/say.view/index.html

```diff
@@ -2,9 +2,9 @@
 <script>
   async function load() {
     let say = await import('./say.js');
-    say.hi(); // Hello!
-    say.bye(); // Bye!
-    say.default(); // Module loaded (export default)!
+    say.hi(); // Olá!
+    say.bye(); // Adeus!
+    say.default(); // Módulo carregado (exportação padrão)!
```

Comment:
Do not translate "export default " to "exportação padrão", as it may cause confusion with the JavaScript keyword. Keep the phrase in English to avoid ambiguity.